pandas/tests/arrays/test_datetimes.py

```diff
@@ -90,3 +90,41 @@ def test_setitem_clears_freq(self):
                                         tz='US/Central'))
         a[0] = pd.Timestamp("2000", tz="US/Central")
         assert a.freq is None
+
+
+class TestReductions(object):
+
+    @pytest.mark.parametrize("tz", [None, "US/Central"])
```

Comment:
in test_reductions?

Reply:
before long, yah.  For now i want to make rebasing 24024 easy on tom